Cargo.toml

```
@@ -49,8 +51,8 @@ tokio = { version = "0.2", features = ["full"] }
 
 [features]
 default = ["http-tls", "ws-tls-tokio"]
-http = ["hyper", "url", "base64"]
-http-tls = ["hyper-tls", "native-tls", "http"]
+http = ["hyper", "hyper-proxy", "url", "base64", "typed-headers"]
+http-tls = ["hyper-tls", "hyper-proxy", "url" ,"native-tls", "http", "typed-headers"]
```

Since, `http` is a required feature of `http-tls` already, these other extra deps don't need to be listed explicitly.

```suggestion
http-tls = ["hyper-tls", "native-tls", "http"]
```